scipy/stats/_covariance.py

```diff
@@ -541,10 +541,14 @@ def __init__(self, cholesky):
         self._factor = L
         self._log_pdet = 2*np.log(np.diag(self._factor)).sum(axis=-1)
         self._rank = L.shape[-1]  # must be full rank for cholesky
-        self._covariance = L @ L.T
         self._shape = L.shape
         self._allow_singular = False
 
+    @cached_property
+    def _covariance(self):
+        return self._factor @ self._factor.T
+
+
```

Comment:
Within a class, PEP8 only requires one blank line between methods.
```suggestion
```

Reply:
Oops I should have including the bit about `[skip ci]` before I pressed the button. The point was that once CI passes, we shouldn't run CI again for deleting a line of whitespace.

Just wanted to make sure you saw this @rishi-kulkarni.

Reply:
Oops, makes sense. I'll keep that in mind.

Reply:
Makes sense - I think my current issue is addressed once `multivariate_t` can take a `Covariance` object, anyway.